lib/features/composer/presentation/composer_controller.dart

```diff
@@ -281,18 +281,6 @@ class ComposerController extends BaseController {
       });
   }
 
-  @override
-  void onError(error) {
-    if (currentOverlayContext != null && currentContext != null) {
-      _appToast.showToastWithIcon(
-          currentOverlayContext!,
-          textColor: AppColor.toastErrorBackgroundColor,
-          message: AppLocalizations.of(currentContext!).message_has_been_sent_failure,
-          icon: _imagePaths.icSendToast);
-    }
-    popBack();
-  }
```

Comment:
Why you delete it? You mean Toast will show unknown error?

Reply:
1. It won't call here. Since we captured `try/catch` it in an `await` from the repository.
2. We also checked the error result from this interactor in the dashboard controller when it had `SendEmailFailure` error

Reply:
got it